scripts/downloads/download_data.py

```diff
@@ -1,54 +1,74 @@
 """Download datasets and specific captures from the datasets."""
 import os
 import shutil
-import sys
+import tarfile
 import zipfile
+from dataclasses import dataclass
 from pathlib import Path
-from typing import Optional
 
 import gdown
 import tyro
 from rich.console import Console
-from typing_extensions import Literal
+
+from nerfstudio.configs.base_config import PrintableConfig
 
 CONSOLE = Console(width=120)
 
 
-def download_blender(save_dir: Path):
-    """Download the blender dataset."""
-    # TODO: give this code the same structure as download_nerfstudio
+@dataclass
```

Comment:
I like these changes, but it should be a separate PR.

Reply:
Shoot, yeah docs probably need to be updated